clang/lib/DPCT/LIBCUAPIMigration.cpp

```diff
@@ -101,5 +101,26 @@ void LIBCURule::runRule(const ast_matchers::MatchFinder::MatchResult &Result) {
   EA.applyAllSubExprRepl();
 }
 
+void LibraryTypeLocRule::registerMatcher(ast_matchers::MatchFinder &MF) {
```

Comment:
please put the code into LibraryAPIMigration.cpp, or create a new.

Reply:
done